docs/synthetic-tokens/long-short-pair.md

```diff
@@ -53,32 +50,69 @@ The method are:
+
+After tokens are minted, to confirm that tokens have been issued by the contract, go to the Read Contract tab of the LSP contract in Etherscan and call the `getPositionTokens` function with the address you minted the tokens from. This will return the number of long and short tokens which will be an equal amount after you mint. The short and long tokens received represent a fully collateralized and risk-neutral position. A minter only receives long or short exposure by selling one of the tokens while a buyer receives exposure simply by purchasing one of the tokens.
 
 ### `redeem`
 
-At any point pre-expiry, anyone can call `redeem` with a pair of long and short tokens and receive an amount of collateral determined  `collateralPerPair`. The total amount of collateral received is determined by `collateralPerPair` * `tokensToRedeem`.
+At any point pre-expiry, anyone can call `redeem` with a pair of long and short tokens and receive an amount of collateral determined by `collateralPerPair`. The total amount of collateral received from the contract is determined by `collateralPerPair` * `tokensToRedeem`.
+
+Regardless of the value of the long and short tokens when redeemed, the summed value of the two tokens will always be worth the total amount of collateral used to mint. The contract burns the tokens and returns collateral proportional to the amount that the token sponsor has deposited to the contract. The caller does not need to approve this contract to transfer any number of redeemed tokens since long and short tokens are burned, rather than transferred.
+
+![](/docs/lsp-tokens/lsp_redeem.png)
 
 ### `expire`
 
-After the expiration timestamp, anyone can call `expire` to request a price from the UMA Optimistic Oracle. This function takes no parameters and will revert if the current time is not later than `expirationTimestamp`. 
+When the current timestamp is later than the `expirationTimestamp` parameter, token holders are unable to settle their tokens for collateral until a price has been received by the Optimistic Oracle. The `settle` function will revert until the `expire` function is called once by anyone. 
+
+The `expire` function does not take any parameters and requests a price from the Optimistic Oracle for the LSP contract's `priceIdentifier`, `expirationTimestamp`, `customAncillaryData`, `collateralToken`, and `prepaidProposerReward`. At this time, a proposal bond can be set associated with the price request using `setBond` along with a custom liveness value for the request using `setCustomLiveness`. Liveness is the amount of time a proposal must wait before being resolved.
+
+![](/docs/lsp-tokens/lsp_expire.png)
 
 ### `settle`
```

Comment:
One other thing I'd add here is the concept of the contract's state. This can be read for the LSP with `contractState`.

Enum values for your context:
- Value of 0: Open (price has not been requested)
- Value of 1: `ExpiredPriceRequested` (one the price has been requested). Small nuance here is that it will still be in this state until the first person calls `settle`, even if a price has been returned to the OO. This is because the LSP has no context that the price has been resolved by the OO until requested successfully.
- Value of 2: `ExpiredPriceReceived` (price was available from OO and first settle call has been performed successfully)

Reply:
Added a section to include.